react/week1/todo-list-app/src/components/Todo.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import './Todo.css'
+function Todo(props) {
+    console.log(props);
+    return (
+        <div>
+            <table>
+            <tr>
+                <td>{props.item.id}</td>
+                <td>{props.item.todo}</td>
+                <td>{props.item.deadline}</td>
+            </tr>
+            </table>
```

Comment:
The issue is here that since Todo is reusable element, therefore it will return a new table each time. 
It makes your HTML look like this in the browser:
```
<div class="mainContainer">
	<div class="headerContainer">
		<h3>My TODO List</h3>
	</div>
	<div>
		<div>
			<table>
				<tr>
					<td>1</td>
					<td>Get out of bed</td>
					<td>Wed Sep 13 2017</td>
				</tr>
			</table>
		</div>
		<div>
			<table>
				<tr>
					<td>2</td>
					<td>Brush teeth</td>
					<td>Thu Sep 14 2017</td>
				</tr>
			</table>
		</div>
		<div>
			<table>
				<tr>
					<td>3</td>
					<td>Eat breakfast</td>
					<td>Fri Sep 15 2017</td>
				</tr>
			</table>
		</div>
	</div>
</div>
```

See it renders several tables, but you need only 1 that wraps around all the table rows. So instead you could just leave the `<tr>` tags in this component and move up the `<table>` tag into the MyList component.